tests/test_files/call.exe

```diff
@@ -0,0 +1 @@
+false
```

Comment:
You might not want to call these test outputs ".exe" as that will look like an executable in windows